geopyspark/geotrellis/rdd.py

```diff
@@ -120,6 +120,32 @@ def to_numpy_rdd(self):
         ser = self.geopysc.create_tuple_serializer(result._2(), value_type=TILE)
         return self.geopysc.create_python_rdd(result._1(), ser)
 
+    def to_tiled_rdd(self, extent=None, layout=None, crs=None, tile_size=256,
```

Comment:
@jpolchlo, @echeipesh and I talked about what information is worth trying to hide from the user - or another way, what information and to what level of detail does the user need to understand underlying mechanics of GPS to be able to use it. 

I think it might be worth thinking about 3 hypothetical users: 

Tier 1 - Someone who has very little programming experience. They are interested in learning more. They have a decent GIS background, and they are trying to take it to the next level because they want to process big raster data sets on the web. 

tier 2 - Okay programmer. Okay GIS background. 

tier 3 - Excellent programmer - understand big data. New to GIS.


Tier 1 is the one I think we need to be particularly careful of not intimidating. Tiers 1 and 2 will be able hack at stuff so long as the documention is there. I hope the same is true for Tier 1 so long as the documentation is good enough.

But. Question for Tier 1 in particular:

1. Is it possible to hide or minimize the fact that they are using Spark? (Is that even a worthwhile goal?) How much do they need to understand what an RDD is? 

Basically - minimizing new concecpts / acronyms is a worthy goal if they don't need to understand them to gain basic benefits from GPS.

On that note - `to_tiled_layer` might make more sense if they dont really need to understand what an RDD is, because a layer (as defined by a set of rasters) is more intuitive.